pom.xml

```
@@ -78,7 +81,7 @@
         <dependency>
             <groupId>org.topbraid</groupId>
             <artifactId>shacl</artifactId>
-            <version>1.1.0</version>
+            <version>1.2.0-SNAPSHOT</version>
```

We should probably just use the systemPath property: https://stackoverflow.com/a/22300875. I've tested this on my local system without performing `mvn install` on the provided .jar file and things appear to work. 
```suggestion
             <version>1.2.0-SNAPSHOT</version>
            <scope>system</scope>
            <systemPath>${project.basedir}/shacl/shacl-1.2.0-SNAPSHOT.jar</systemPath>
```

---

As @ecurley-nextcentury and I have discussed offline, the use of `<systemPath>` is deprecated by maven.  See [this](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#System_Dependencies) and [this](https://issues.apache.org/jira/browse/MNG-6523) for more info.  We both saw it mentioned in a [StackOverflow article](https://stackoverflow.com/questions/4955635/how-to-add-local-jar-files-to-a-maven-project) (see second answer), but I didn't use it because it was deprecated.

That said, I'm concerned that anybody using this validator will probably not see the change in the README, and things will just stop working for them.  So I'm fine with using it, but what do you think, @craigwarsaw ?

---

This has elicited further discussion that will resume with the entire team (or at least those that work with AIF).  In short, the thought is to have a branch within AIF which serves as the "AIF Developer Edition" that uses the TopBraid 1.2 snapshot, contains fail-fast and the progress monitor (AIF-791).  Then we'd keep AIF master using 1.1, with no abort flag or progress monitor.

---

Based on discussions, once we tag AIF Version 1.0, we'll merge this PR into master, along with the deprecated use of `<systemPath>`.

---

Actually, I don't think we should use a tag for 1.0.  The master branch should always have the latest version of code.  Release 1.0 of AIF would be a separate long-lived branch (until we decide to no longer support it) using TopBraid 1.1.  We would have to maintain each release branch.  Fixes on the AIF 1.0 branch would have to be cherry-picked to their own feature branch off of master.  Check out this article https://docs.microsoft.com/en-us/azure/devops/repos/git/git-branching-guidance?view=azure-devops

Documenting this in AIDA Collaboration and making the AIF 1.0 branch would be part of AIDA-801.

---

It turns out that `<systemPath>` doesn't actually install the jar in the `target` directory, so the validator doesn't execute.  Instead, we now create a local maven repository at `./repo` and refer to in `pom.xml`.  A Readme file was added to to `./repo`.